scripts/variantstore/wdl/GvsExtractCallset.wdl

```diff
@@ -182,7 +185,7 @@ task ValidateFilterSetName {
 
     echo "project_id = ~{query_project}" > ~/.bigqueryrc
 
-    OUTPUT=$(bq --location=US --project_id=~{query_project} --format=csv query --use_legacy_sql=false "SELECT filter_set_name as available_filter_set_names FROM ~{data_project}.~{data_dataset}.filter_set_info GROUP BY filter_set_name")
+    OUTPUT=$(bq --location=US --project_id=~{query_project} --format=csv query --use_legacy_sql=false "SELECT filter_set_name as available_filter_set_names FROM \`~{data_project}.~{data_dataset}.filter_set_info\` GROUP BY filter_set_name")
```

Comment:
Were you able to run this version successfully? This doesn't execute without this change for me.

Reply:
FWIW, here is the error I got:
```
sdk@sha256:ff4546d0bab6048b4ae61ddfb1dfdccb12b3725f5833fb696a5c5915e5bcdd15 /cromwell_root/script
+ '[' false = true ']'
+ echo 'project_id = terra-vpc-sc-dev-d59ab2f1'
++ bq --location=US --project_id=terra-vpc-sc-dev-d59ab2f1 --format=csv query --use_legacy_sql=false 'SELECT filter_set_name as available_filter_set_names FROM fc-aou-cdr-synth-test-2.1kg_wgs_2022q1.filter_set_info GROUP BY filter_set_name'
+ OUTPUT='Error in query string: Error processing job '\''terra-vpc-sc-
dev-d59ab2f1:bqjob_r6cfbf771954bf621_00000181043dd904_1'\'': Syntax error: Missing
whitespace between literal and alias at [1:84]'
2022/05/27 06:39:34 Starting delocalization.
```

Reply:
Nice catch!